part-spec/definitions.json

```diff
@@ -186,164 +186,165 @@
-          "comment": "mostly commonly power or ground pin",
-          "type": "string"
-        },
-        "dependencies": {
-          "componentUnit": [
-            "minValue",
-            "maxValue",
-            "value"
-          ]
-        }
+      },
+      "componentUnit": {
+        "description": "unit of min/max value",
+        "type": "string",
+        "enum": [
+          "ohms",
+          "kilo ohms",
+          "mega ohms",
+          "micro farads",
+          "pico farads",
+          "micro henries"
+        ]
+      },
+      "connectionPin": {
+        "description": "name of pin to which an external component should be pulled up",
+        "comment": "mostly commonly power or ground pin",
+        "type": "string"
+      },
+      "dependencies": {
+        "componentUnit": [
+          "minValue",
+          "maxValue",
+          "value"
+        ]
       }
-    },
-    "pinFunction": {
-      "type": "object",
-      "properties": {
-        "perFunctionName": {
-          "description": "name of the function of a pin",
-          "examples": [
-            "UART5_TX",
-            "SPI3_MOSI",
-            "PC12"
-          ],
-          "type": "string"
-        },
-        "interfaceType": {
-          "description": "type of interface enabled by pin",
-          "examples": [
-            "i2c",
-            "spi",
-            "uart",
-            "usb2",
-            "usb3",
-            "usb4",
-            "pcie",
-            "sdio",
-            "memory",
-            "emmc",
-            "annode",
-            "cathode"
-          ],
-          "type": "string"
-        },
-        "pinUsage": {
-          "description": "standardized usage of pin",
-          "examples": [
-            "UART_TX",
-            "UART_RX",
-            "SPI_COPI",
-            "SPI_CLK",
-            "SPI_CS",
-            "SPI_CIPO",
-            "I2C_SCL",
-            "I2C_SDA",
-          ],
-          "type": "string"
-        },
-        "direction": {
-          "description": "direction of a pin's function",
-          "enum": [
-            "in",
-            "out",
-            "bidir",
-          ],
-          "type": "string"
-        },
-        "electricalConfiguration": {
-          "description": "electrical configuration of a pin",
-          "examples": [
-            "open-drain",
-            "push-pull",
-            "analog",
-            "power",
-            "ground",
-            "high-impedance"
-          ],
-          "type": "string"
-        },
-        "polarity": {
-          "description": "whether the active state of a pin is high or low",
-          "type": "string",
-          "enum": [
-            "high",
-            "low"
-          ]
-        }
+    }
+  },
+  "pinFunction": {
+    "type": "object",
+    "properties": {
+      "perFunctionName": {
+        "description": "name of the function of a pin",
+        "examples": [
+          "UART5_TX",
+          "SPI3_MOSI",
```

Comment:
Line 223 - Change "SPI_MOSI" To be consistent with SPI_COPI notation below

Reply:
done.